.github/workflows/extension-release-published.yml

```diff
@@ -146,7 +153,7 @@ jobs:
                       -Dclassifiers=,sources,javadoc,
 
   deploy_xsd:
-    if: inputs.nameSpace != ''
+    if: inputs.nameSpace != '' && inputs.deployToMaven == true
```

Comment:
@sayaliM0412 @jandroav @filipelautert Just confirming here: Are published XSDs only required if something is going to maven central?

Reply:
Makes sense. All the libraries that we provide that also provides an XSD are uploaded to maven. And anyways the XSD should be shipped with the jar file, so seems correct for me.